packages/server/graphql/mutations/startRetrospective.ts

```diff
@@ -61,7 +61,7 @@ export default {
       meetingType,
       dataLoader
     )
-    const team = await getTeamByTeamId(teamId)
+    const team = await getTeamsById([teamId])[0]
```

Comment:
be defensive! if they pass in "foo" as the teamId this will be undefined & throw.